client/src/components/Home/AboutUs/Event/Event.js

```diff
@@ -3,56 +3,52 @@ import axios from "axios";
 import "bootstrap/dist/css/bootstrap.min.css";
 import planet from "./planet.svg";
 import mobilePlanet from "./mobile-planet.svg";
-import "./Notice.css";
+import "./Event.css";
 
-function Notice() {
-  const [notices, setNotice] = useState([
-    { title: "fetching notices", description: " . . ." },
+function Event() {
+  const [events, setEvent] = useState([
+    { title: "Recieving signals...", description: "Decrypting... Latest News..." },
   ]);
   useEffect(() => {
     axios
-      .get("/api/notice")
+      .get("api/event")
       .then((res) => {
         const { result } = res.data;
-        setNotice([...result]);
+        setEvent([...result]);
       })
       .catch((err) => {
         const result = [];
         result.push({
           title: "Something went wrong!",
           description: "Try reloading. . .",
         });
-        setNotice(result);
+        setEvent(result);
       });
   }, []);
 
   return (
-    <div className="notice-section">
-      <div className="notice-title">From the Geeks' Satellite..</div>
-      <div className="notice-content">
-        <div className="title">
-          Recieving signals...{" "}
-          <span className="decrypting"> Decrypting...</span>
-          <span className="latest-news"> Latest News...</span>
```

Comment:
I think we should maintain this structure.. and replace "Latest News..." with "Latest Events.."